Program.cs

```diff
@@ -219,59 +216,78 @@ private static void FullServer()
             }).ConfigureAwait(false);
 
             var mode = SimplerConfig.Config.Instance["MODE"];
+            var modes = SimplerConfig.Config.Instance["MODES"]?.Split(",");
+            if (modes == null)
+                modes = new string[] { "indexer", "updater", "flipper" };

```

Comment:
const vars / enums ?

Reply:
like that https://github.com/Coflnet/HypixelSkyblock/pull/40/files#diff-bfa1af4d3e2db26d49d3b197d78b22e7cfd5163332454c9fb58bb0c0463349c6R25